docs/spec-update.md

```diff
@@ -1,7 +1,55 @@
 ---
 id: spec-update
-title: spec-update
-sidebar_label: spec-update
+title: Update Operation
 ---
 
-spec-update TODO
+## Update Specification
+
+Update operation is enabled accepting single inputType as arguments
+
+For `Note` type:
+```graphql
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  comments: [Comment]!
+}
+```
+
+Following mutation will be created:
+```graphql
+type Mutation {
+    updateNote(input: MutateNoteInput!): Note!
+}
+```
+
+Input type for update operation looks as follows
+```graphql
+input MutateNoteInput {
+  # ID field is required for update
+  id: ID!
+  title: String
+  description: String
+}
+```
+
+## Variations
+
+### Contidional updates
+
+Conditional updates can be enabled for the cases where we want to perform update 
+operation only after meeting certain criteria
+
+```graphql
+type Mutation {
+  updateNote(input: MutateNoteInput!, where: UpdateNoteFilter): Note!
```

Comment:
Where does the `where: UpdateNoteFilter ` coming from? 

Is this the same as the `filter` here https://github.com/graphqlcrud/spec/pull/6/files#diff-6c9e4cac1fb75081a4efb6eaa188da54R67?

Reply:
We do not have this implemented but I have put Filter example bellow.

Reply:
Okay thanks.